Source/WebGPU/WGSL/Parser.cpp

```diff
@@ -77,6 +77,12 @@ struct TemplateTypes<TT> {
         return { WTFMove(astNodeResult) }; \
     } while (false)
 
+#define RETURN_ARENA_NODE(type, ...) \
```

Comment:
I don't love the naming here, but `RETURN_NODE_REF` was already taken. We can rename later once we refactor all the nodes.